docs/content/opa-1.md

```diff
@@ -0,0 +1,201 @@
+---
+title: OPA 1.0
+kind: misc
+weight: 4
+---
+
+OPA v1.0 will introduce breaking changes to the Rego language. 
```

Comment:
There are also server changes such as `Binding the OPA server to the `localhost` interface by default`.


I know of at least one issue to the OPA server labelled as v1: https://github.com/open-policy-agent/opa/issues/2137 - there might be others too. Does it make sense to talk more generally about breaking changes with this in mind?